### PR TITLE
feat(forms): shadows on form inputs

### PR DESCRIPTION
**Pull request by aredridel**

### CODE REVIEW

**Comment by aredridel** [comment]
Based on rounded corners PR for now; will rebase as that is merged.

**Comment by aredridel** [comment]
screen shot 2016-05-19 at 16 21 43
screen shot 2016-05-19 at 16 21 21

Differences from design (right): 
- smaller text on help block, as design type is between font-size-small and default type size 
- smaller text in input, as design type is between font size default and font-size-medium.

**Comment by aredridel** [comment]
Hover/active/focus state:

screen shot 2016-05-19 at 16 25 43

**Comment by aredridel** [comment]
Needs review of https://github.com/npm/pivotal-ui/pull/169/commits/0a1e6ee71c525850f13469f06f98695577c844a3 onward.

**Comment by aredridel** [comment]
The form interactions:

form interaction

**Comment by gabrrra** [comment]
Looks good, thanks @aredridel !

**Comment by jefflembeck** [comment]
On first review, these look fantastic. Combing through more now

File: src/pivotal-ui/components/forms/forms.scss [hunk 1]
```diff
@@ -557,16 +557,19 @@ them over a custom dropdown whenever possible.
...
   position: relative;
   &:hover, &:focus {
     border-color: $input-border-color-hover;
   }
 
-  &.active, &:active {
-    background-color: $input-bg-active;
+  font-weight: 400;
+
+  box-shadow: inset 0 0px 20px rgba(127, 127, 127, 0.2);
```
Comment (jefflembeck) [inline src/pivotal-ui/components/forms/forms.scss]: Is this worth making a mixin? (wondering because of button-shadow above)

**Comment by jefflembeck** [comment]
Other than the comment above, i'm all about 🐑

**Comment by jefflembeck** [comment]
Everything looks good except for these buttons.

pivotal ui css forms 2016-06-01 16-03-00

And the validation section (which doesn't seem to have any of the new styles applied)

**Comment by gabrrra** [comment]
Can we take the shadows off the +/- buttons?

**Comment by jefflembeck** [comment]
Fixed last issue. Merging.

**Pull request merged**